services/algolia-sync/src/index.js

```diff
@@ -0,0 +1,45 @@
+const yargs = require('yargs');
+const recieveMessage = require('./components/aws-sqs/sqs');
+const siteSync = require('./components/algolia/site-sync');
+
```

Comment:
For safety, you might want to force node to throw a hard error on any unhandled promise rejection. If so, place this code block after your `require` statements:

```js
process.on('unhandledRejection', (e) => {
  throw e;
});
```

Also, if you plan to add new relic at some point, you should also include a `newrelic.noticeError(e);` call inside that function as well (before the error is thrown)

Reply:
Sounds good, I just added Newrelic and added this in like you asked.